go/internal/initutil/ipfs.go

```diff
@@ -46,6 +46,7 @@ func (m *Manager) SetupLocalIPFSFlags(fs *flag.FlagSet) {
 	fs.StringVar(&m.Node.Protocol.Tor.Mode, "tor.mode", defaultTorMode, "if true tor will be enabled")
 	fs.StringVar(&m.Node.Protocol.Tor.BinaryPath, "tor.binary-path", "", "if set berty will use this external tor binary instead of his builtin one")
 	fs.BoolVar(&m.Node.Protocol.DisableIPFSNetwork, "p2p.disable-ipfs-network", false, "disable as much networking feature as possible, useful during development")
+	fs.BoolVar(&m.Node.Protocol.RelayHack, "p2p.relay-hack", false, "if set berty will force libp2p to use randoms relays from the config and will advertise as ours optimisticaly")
```

Comment:
```suggestion
	fs.BoolVar(&m.Node.Protocol.RelayHack, "p2p.relay-hack", false, "*temporary flag*; if set, Berty will only advertise relayed addresses")
```

Reply:
> Berty will only advertise relayed addresses

It will not stop advertising other addresses (we want to have as much chances as possible).
Maybe `*temporary flag*; shouldn't be set` would be good ?